### ISSUE TITLE
wow classic beta error

### ISSUE THREAD

**Issue by mooreatv** [mooreatv/WhoTracker#1]
Wow classic beta error
```
Message: Interface\AddOns\WhoTracker\WhoTracker.lua:253: attempt to call global 'SetWhoToUI' (a nil value)
Time: Thu May 23 00:53:23 2019
Count: 1
Stack: Interface\AddOns\WhoTracker\WhoTracker.lua:253: attempt to call global 'SetWhoToUI' (a nil value)
Interface\AddOns\WhoTracker\WhoTracker.lua:253: in function `SendWho'
Interface\AddOns\WhoTracker\WhoTracker.lua:203: in function <Interface\AddOns\WhoTracker\WhoTracker.lua:198>

Locals: friendsFrame = FriendsFrame {
 0 = <userdata>
 TitleText = FriendsFrameTitleText {
 }
 portrait = FriendsFramePortrait {
 }
 LeftBorder = FriendsFrameLeftBorder {
 }
 FriendsTabHeader = FriendsTabHeader {
 }
 TopRightCorner = FriendsFrameTopRightCorner {
 }
 TopLeftCorner = FriendsFrameTopLeftCorner {
 }
 Bg = FriendsFrameBg {
 }
 playerStatusFrame = 1
 TopBorder = FriendsFrameTopBorder {
 }
 selectedIgnore = 1
 TopTileStreaks = FriendsFrameTopTileStreaks {
 }
 TitleBg = FriendsFrameTitleBg {
 }
 BottomBorder = FriendsFrameBottomBorder {
 }
 selectedFriend = 1
 OnLoad = <function> defined @Interface\SharedXML\SharedUIPanelTemplates.lua:860
 numTabs = 4
 BotRightCorner = FriendsFrameBotRightCorner {
 }
 Inset = FriendsFrameInset {
 }
 CloseButton = FriendsFrameCloseButton {
 }
 BotLeftCorner = FriendsFrameBotLeftCorner {
 }
 PortraitFrame = FriendsFramePortraitFrame {
 }
 selectedTab = 1
 RightBorder = FriendsFrameRightBorder {
 }
}
(*temporary) = nil
(*temporary) = 1
(*temporary) = "attempt to call global 'SetWhoToUI' (a nil value)"
```

**Comment by mooreatv** [comment]
Use C_FriendList.SetWhoToUi instead
 SetWhoToUI = C_FriendList.SetWhoToUi;

**Comment by mooreatv** [comment]
fixed in 2.1.0 (but can't test the fix as I don't have beta access)
